plugins/extensions/plugins/DeflectPlugin.cpp

```diff
@@ -225,7 +226,9 @@ void DeflectPlugin::_handleDeflectEvents(Engine& engine,
         case deflect::Event::EVT_VIEW_SIZE_CHANGED:
         {
             Vector2ui newSize(event.dx, event.dy);
-            if (engine.getCamera().getType() == CameraType::stereo)
+            const auto isStereo =
+                _renderingParams.getStereoMode() == StereoMode::side_by_side;
```

Comment:
camera.getStereoMode() looks still more right to me, no?

Reply:
Dito